From d3864018ef09ef38e1733110e2643735067bc02f Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Sun, 6 Jan 2013 14:49:40 -0500
Subject: [PATCH] use newer config headers macro

With newer automake, it throws an error:
configure.ac:5: error: 'AM_CONFIG_HEADER': this macro is obsolete.
    You should use the 'AC_CONFIG_HEADERS' macro instead.

The newer macro has been around a long time, so there's no need to
worry about backwards compat here.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index ebf657f..9980c5a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -2,7 +2,7 @@ AC_PREREQ([2.57])
 AC_INIT(evtest,[1.30], [https://bugzilla.freedesktop.org/enter_bug.cgi?product=evtest], evtest)
 AM_INIT_AUTOMAKE([foreign dist-bzip2])
 
-AM_CONFIG_HEADER(config.h)
+AC_CONFIG_HEADERS(config.h)
 AC_PROG_CC
 AC_PROG_INSTALL
 
-- 
1.8.0.2

